.eslintrc.js

```diff
@@ -11,6 +11,15 @@ module.exports = {
         templateFile: 'src/licenseHeader.js',
       },
     ],
+    'import/no-extraneous-dependencies': LEVEL,
   },
+  overrides: [
+    {
+      files: ['**/*.test.js', '**/__mocks__/**'],
+      rules: {
+        'import/no-extraneous-dependencies': 'off',
```

Comment:
What problem was this causing?

Reply:
In our tests we rely on the testing environment of the root project, not the one of the current package. So this plugin was reporting some false positives because it doesn't know about this.